### ISSUE TITLE
Stamping paper is buggy as hell.

### ISSUE THREAD

**Issue by Owai-Seek** [Skyrat-SS13/Skyrat13#3097]
Issue reported from  Round ID: 1926 (Skyrat SS13 - MRP-HRP [Apply on Discord, Weekly Events])

Reporting client version: 513

Attempt to stamp a paper at cargo.

Attempt to stamp where it tells you.

Stamp overlay constantly drags itself to the top of the paper.

**Comment by Investigator77** [comment]
Move your mouse slowly.

**Comment by YakumoChen** [comment]
Can't you just stamp the paper icon and it will automatically put a stamp on the bottom?

**Comment by Owai-Seek** [comment]
Nope, it forces you to do this now, and moving slowly doesn't work either. It pulls it to the top either way.
